api/user/schema.py

```diff
@@ -216,6 +215,8 @@ class Arguments:
 
     @Auth.user_roles('Admin')
```

Comment:
please add the `Super_Admin` user role here as super admin should be ab;e to perform this task